ckanext/ontario_theme/templates/internal/snippets/search_form.html

```diff
@@ -47,166 +50,213 @@ <h2 class="ontario-label">
-                          <svg class="ontario-icon"
-                              focusable="false"
-                              sol:category="primary"
-                              viewBox="0 0 24 24"
-                              preserveAspectRatio="xMidYMid meet">
-                            <use xlink:href="#ontario-icon-close"></use>
-                          </svg>
-                        </span>
-                      </a>
-                      {%- if loop.index == 1 -%}</span>{%- endif -%}
+  <!-- block for package searches -->
+  {% block search_facets %}
+    {% if facets %}
+      <p class="filter-list">
+        {% if query %}
+          {#
+      Groups/Organizations don't have facets set or the
+      remove_field() function in their index() action.
+      Only show when facets and a query are available to avoid
+      missing attribute error.
+      #}
+        {% endif %}
+        {% for field in facets.fields %}
+          {% if field != 'access_level' %}
+            <span class="facet-row">
+              {% set search_facets_items = facets.search.get(field)['items'] %}
+              <span class="facet">
+                <span class="nowrap">
+                  {% if (not query and not loop.index0==0) or query %}<span class="sr-only">and</span>{% endif %}
+                  {{ facets.titles.get(field) + _(':') if facets.titles.get(field) is not none }}
+                  {% for field_value in facets.fields[field] %}
+                    {% set dict = h.get_facet_items_dict(field, limit=limit)|selectattr("name", "equalto", field_value)|list if h.get_facet_items_dict(field, limit=limit) %}
+                    {% set item = dict[0] if dict else ({'name': field_value, 'display_name': field_value }) %}
+                    {{ h.snippet('snippets/translate_facets.html', fields=schema_dict, name=field, item=item) }}
+                    {% set item_count = h.localised_number(item['count']) if item['count'] else 0 %}
+                    <a href="{{ facets.remove_field(field, item.name) }}"
+                       class="filtered pill"
+                       title="{{ _('Remove') }}">
+                      <span class="pill-contents">
+                        {{ item.display_name }} ({{ item_count }})
+                        <svg class="ontario-icon"
+                             focusable="false"
+                             sol:category="primary"
+                             viewBox="0 0 24 24"
+                             preserveAspectRatio="xMidYMid meet">
+                          <use xlink:href="#ontario-icon-close"></use>
+                        </svg>
+                      </span>
+                    </a>
+                    {%- if loop.index == 1 -%}</span>{%- endif -%}
                     {% endfor %}
+                  </span>
                 </span>
-              </span>
+              {% endif %}
             {% endfor %}
             {% if c.fields|length >= 2 %}
```

Comment:
Make it `>= 3` since there is always the `access_level` element in the `c.fields` array that will count as 1, e.g.:

![Screenshot 2024-09-18 at 16 23 06](https://github.com/user-attachments/assets/8c9bd838-86dc-4846-a7cc-2ef0cbcbd9e1)

Reply:
This doesn't solve the problem, as you can have multiple access levels checked and `c.fields|length` will increase. All 3 access levels checked will have the same effect (The displaying of the button is all done in the javascript). I've opted to removing the if statement and leaving the javascript as if.